src/Resources/app/storefront/src/js/filter-category-select.plugin.js

```diff
@@ -16,18 +17,38 @@ export default class FilterCategorySelectPlugin extends FilterBasePlugin {
 
     init() {
         this.selection = [];
+        this.showChecked();
         this.counter = DomAccess.querySelector(this.el, this.options.countSelector);
         this._registerEvents();
+
```

Comment:
@D3xter7 please remove the extra line here, and similar other places in the file (especially in the starting and ending of functions). You can separate the code within a function with a single line for better readability.